pdf_melted_app/src/pages/index.js

```diff
@@ -1,13 +1,12 @@
 import React from "react"
 import { Link } from "gatsby"
-import PdfSample from "./pdf_documents/pdf_sample.pdf" // Tell Webpack this JS file uses this PDF
```

Comment:
You don't want to import via Webpack like this. If you create PDFs where the path is known at build time--you can just rely on that path existing.